src/main/kotlin/racingcar/RacingCarGamePlay.kt

```diff
@@ -0,0 +1,30 @@
+package racingcar
+
+import racingcar.domain.RacingCar
+import racingcar.domain.RacingCarGame
+import racingcar.ui.InputView
+import racingcar.ui.ResultView
+import racingcar.util.RandomNumGenerator
+
+fun main() {
+    val racingCarNames = InputView.inputNames()
+    val countOfRacing = InputView.inputCount()
+
+    ResultView.printInitMessage()
+    val randomNumGenerator = RandomNumGenerator()
+    var currentRacingCarGame = RacingCarGame.init(initRacingCars(racingCarNames))
```

Comment:
객체를 생성하는 시점과 사용하는 시점은 가급적 가까운 것이 좋은 것 같아요. 😃 
[클린코드 - 형식 맞추기](https://github.com/Yooii-Studios/Clean-Code/blob/master/Chapter%2005%20-%20%ED%98%95%EC%8B%9D%20%EB%A7%9E%EC%B6%94%EA%B8%B0.md#%EB%B3%80%EC%88%98%EC%84%A0%EC%96%B8)

```suggestion
    var currentRacingCarGame = RacingCarGame.init(initRacingCars(racingCarNames))
    val randomNumGenerator = RandomNumGenerator()
```